UltraStar Play/Assets/Editor/SceneList/SceneListEditorWindow.cs

```diff
@@ -45,6 +54,12 @@ void OnGUI()
             }
             EditorGUILayout.EndScrollView();
         }
+        catch (ArgumentException)
+        {
+            // Unity always throws an exception after a restart:
```

Comment:
Please log this exception + stacktrace. In general, never just catch an exception silently and do nothing.